gen_pools_and_gauges.py

```diff
@@ -94,8 +129,30 @@ def main():
         )
         if result:
             gauges[chain] = result
-        with open("extras/gauges.json", "w") as f:
-            json.dump(gauges, f, indent=2)
+
+        # core pools
+        if chain in ["sepolia", "goerli"]:
+            continue
+        core_pools[chain] = BalPoolsGauges(chain).core_pools
+
+        # cache mainnet BalPoolsGauges
+        if chain == "mainnet":
+            gauge_info_mainnet = gauge_info
+
+    # root gauges; only on mainnet
+    result = process_query_root_gauges(gauge_info_mainnet.query_root_gauges(), gauges)
+    if result:
+        root_gauges["mainnet"] = result
+
```

Comment:
extras are normally user inputs and outputs go in outputs. Just noticed this now, but it's a bit confusing because in most cases extras/ are user defined things and outputs are generated things have a different file per chain. It seems like it would make sense to align this, and I think a number of things are already using the current output format. Thoughts?

Reply:
Also I thought we were going to break core pools into a separate action with it's own delta, but it seems to me like this doesc ore pools in the pools_gauges run and deletes the .py file that did core pools.   Don't we want a separate script and a separate action/branch/pr?

Reply:
c295c18

Reply:
> Also I thought we were going to break core pools into a separate action with it's own delta, but it seems to me like this doesc ore pools in the pools_gauges run and deletes the .py file that did core pools. Don't we want a separate script and a separate action/branch/pr?

for this, i indeed batched core pools generation with the other pools and gauges generation. _that_ batch however, is now a separate action (`generate_pools_gauges.yaml`), independent of the general address book generation.

i think seeing the diffs on both all new pools and core pools could be useful actually, no?

Reply:
Pools there are tons of, it is very noisy and permissionless. 

Core pools are permissioned and very important.  For this reason I think it is important that we separate out the core pools deltas by themselves adn have a much more detailed review process there.

Reply:
0aa4b22